github/deploy.py

```diff
@@ -89,8 +89,18 @@ def get_github_token():
 # So if we have a folder within a folder, both conditions are
 # satisfied and we're able to proceed
 dummy_directory = tempfile.mkdtemp(dir=directory_to_upload)
+
 for fl in targets:
-    shutil.copy(fl, os.path.join(directory_to_upload, os.path.basename(fl)))
+    if fl.endswith('zip'):
+        extension = 'zip'
+    elif fl.endswith('tar.gz'):
+        extension = 'tar.gz'
+    else:
+        raise ValueError('This file is neither a zip nor a tar.gz: {}'.format(fl))
```

Comment:
@lolski given `bazel` limits what we pass it targets, is it even possible to hit this?

Reply:
@lolski also, a better way to split out extension is `os.path.splitext`

Reply:
As we've discussed in Slack,

#1 I think it's better to keep the `else` bit since the information regarding the allowed extension being `zip` and `tar.gz` is in a different file and therefore is too far removed

#2 won't work because for example, it will split `grakn-core-all.tar.gz` into `grakn-core-all.tar` + `.gz` rather than `grakn-core-all` + `.tar.gz`.